maquetado/setup/index.html

```diff
@@ -8,12 +8,65 @@
   <body>    
-    <h2 class="title">Front-End Tools</h2>                
-    <div class="tech-front-container">
-      <img class="tech-img" alt="React Logo" src="img/react_icon.svg" />
-      <h4 class="tech-title">React</h1>
-      <h5 class="tech-release">Release: 2013</h2>
-      <p class="tech-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
-    </div>  
+    <h2 class="title">Front-End Tools</h2>    
+    <div class="tech-catalog">
+      <div class="tech-front-container">
+        <img class="tech-img" alt="React Logo" src="img/react_icon.svg" />
+        <h4 class="tech-title">React</h1>
+        <h5 class="tech-release">Release: 2013</h2>
+        <p class="tech-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+      </div>
+      <div class="tech-front-container">
+        <img class="tech-img" alt="AngularJS Logo" src="img/angular_icon.svg" />
+        <h4 class="tech-title">AngularJS</h1>
+        <h5 class="tech-release">Release: 2013</h2>
+        <p class="tech-description">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.</p>
+      </div>
+      <div class="tech-front-container">
+        <img class="tech-img" alt="VueJS Logo" src="img/vue_icon.svg" />
+        <h4 class="tech-title">VueJS</h1>
+        <h5 class="tech-release">Release: 2014</h2>
+        <p class="tech-description">Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces.</p>
+      </div>
+      <div class="tech-front-container">
+        <img class="tech-img" alt="Elm Logo" src="img/elm_icon.svg" />
+        <h4 class="tech-title">Elm</h1>
+        <h5 class="tech-release">Release: 2016</h2>
+        <p class="tech-description">Elm is a domain-specific programming language for declaratively creating web browser-based graphical user interfaces.</p>
+      </div>
+      <div class="tech-front-container">
+        <img class="tech-img" alt="Polymer Logo" src="img/polymer_icon.svg" />
+        <h4 class="tech-title">Polymer</h1>
+        <h5 class="tech-release">Release: 2015</h2>
+        <p class="tech-description">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+      </div>
+      </div>
+      <h2 class="title">Back-End Tools</h2>  
+      <div class="tech-catalog">
+      <div class="tech-front-container">
+        <img class="tech-img" alt="Ruby on Rails Logo" src="img/rails_icon.png" />
+        <h4 class="tech-title">Ruby on Rails</h1>
+        <h5 class="tech-release">Release: 2005</h2>
+        <p class="tech-description">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+      </div>
+      <div class="tech-front-container">
```

Comment:
This techs are not `front`, what do you think of changing the class name for one more descriptive?